### PR TITLE
Secure submit to GlobalPayment JS migration

### PR DESCRIPTION
**Pull request by mayur-vachchewar**

### CODE REVIEW

File: admin/controller/extension/payment/securesubmit.php [hunk 1]
```diff
@@ -155,12 +155,7 @@ public function index() {
 		} else {
 			$data[$prefix . 'securesubmit_total'] = $this->config->get($prefix . 'securesubmit_total');
 		}
-		
-		if (isset($this->request->post[$prefix . 'securesubmit_use_iframes'])) {
-			$data[$prefix . 'securesubmit_use_iframes'] = $this->request->post[$prefix . 'securesubmit_use_iframes'];
-		} else {
-			$data[$prefix . 'securesubmit_use_iframes'] = $this->config->get($prefix . 'securesubmit_use_iframes');
-		}
+
```
Comment (MSmedal) [inline admin/controller/extension/payment/securesubmit.php:158]:
```suggestion
```

File: catalog/view/stylesheet/securesubmit.css [hunk 2]
```diff
@@ -41,11 +52,13 @@
...
     text-transform:uppercase;
+    position: relative;
+    left: 1%;
 }
 
 #hpsPaymentForm .form-horizontal .control-label {
     text-align:left;
-    width:60%; 
+    width:60%;
     padding-left:none;
```
Comment (MSmedal) [inline catalog/view/stylesheet/securesubmit.css]: Not an issue you introduced, but can you fix this? ['none' isn't a value of padding-left](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-left)

File: catalog/view/theme/default/template/extension/payment/securesubmit.tpl
Comment (MSmedal) [inline catalog/view/theme/default/template/extension/payment/securesubmit.tpl:1]: I'm having some system issues when I try to spin up an OpenCart 2.3 site to run this one. Could you upload a snippet of the card info input fields we're creating for my reference?

**Review by MSmedal — CHANGES_REQUESTED** [review]
Looks great! Just a bit of housekeeping to finish

File: catalog/view/theme/default/template/extension/payment/securesubmit.tpl
Comment (mayur-vachchewar) [reply catalog/view/theme/default/template/extension/payment/securesubmit.tpl:1]: > I'm having some system issues when I try to spin up an OpenCart 2.3 site to run this one. Could you upload a snippet of the card info input fields we're creating for my reference?

Here is the reference:

Comment (mayur-vachchewar) [reply catalog/view/theme/default/template/extension/payment/securesubmit.tpl:1]: image

**Review by MSmedal — APPROVED** [review]

**Pull request merged**